feed-integration-webapp/src/main/resources/db/changelog/liquibase.xml

```diff
@@ -0,0 +1,50 @@
+<databaseChangeLog logicalFilePath="liquibase.xml"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+    <property name="schemaName" value="crater_atomfeed"/>
+    <include file="classpath:sql/db_migrations.xml"/>
+
+    <changeSet id="1" author="Bahmni" >
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="event_records_offset_marker"/>
+            </not>
+        </preConditions>
+        <createTable tableName="event_records_offset_marker">
+            <column name="id" type="int" autoIncrement="true">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="event_id" type="int"/>
+            <column name="event_count" type="int"/>
+            <column name="category" type="varchar(255)"/>
+        </createTable>
+    </changeSet>
+
+    <changeSet id="2" author="Bahmni">
+        <comment>Creating quartz cron scheduler Table</comment>
+        <createTable tableName="quartz_cron_scheduler">
+            <column name="id" type="int" autoIncrement="true">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="name" type="varchar(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="enabled" type="boolean" defaultValue="true">
+            </column>
+            <column name="cron_statement" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="start_delay" type="int">
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet id="7" author="Bahmni">
+        <comment>Adding schedulers for openmrs patient</comment>
+        <sql>
+            insert into quartz_cron_scheduler (name, cron_statement, start_delay, enabled) values ('openMRSPatientFeedJob', '0/15 * * * * ?', 0, true);
```

Comment:
Suggest to rename this cron job to: `openmrsPatientFeedForCraterJob` to ensure its clear why this cron job is registered in the system.

Reply:
It should ideally run every 2 mins. 15 mins seems to long a time. For clinics we don't expect too much foot fall, so the scheduler should finish quite quickly. We increase this time only if this impacts system performance.

Reply:
The cron job is currently running every 15secs, not 15 mins. Let me know if it should run every 2 mins instead. Name of cron job has been changed.